### DISCUSSION TITLE
Players.json for TMDB helper plugin

### DISCUSSION THREAD

**Discussion by TemofeyF** [quarckster/kodi.kino.pub#203] [Q&A]
Доброго времени суток! Я пытаюсь сделать json файл для TMDB helper. Пока что получилось только вот это:
```json
{
    "name": "[COLOR FFFF6B00]KinoPub[/COLOR]",
    "plugin": "video.kino.pub",    
    "priority": 100,    
    "is_resolvable": "true",
    "play_movie" : [
        "plugin://video.kino.pub/items/movies/alphabet/A/",
        {"title": "{title}", "year": "{year}"}
    ]
}
```
Естественно ищет только фильмы на букву "А"))). Подскажите какой путь для всех фильмов/сериалов?
А большинство "забугорных плагинов ищет по imdb_id, tmdb_id, trakt_id. Пример:
plugin://plugin?info=play&amp;tmdb_id=$INFO[ListItem.Property(tmdb_id)]&amp;type=movie)
Подскажите пожалуйста, что нужно сделать, чтобы это заработало?

**Comment by quarckster** [comment]
Для поиска надо использовать следующий url: `plugin://video.kino.pub/search/<тип контента>/results?title=<имя фильма/сериала>`. Место в коде https://github.com/quarckster/kodi.kino.pub/blob/master/src/resources/lib/main.py#L228

Для воспроизведения: `plugin://video.kino.pub/play/<id фильма/сериала>?season_index=<номер сезона>&index=<номер эпизода>` https://github.com/quarckster/kodi.kino.pub/blob/master/src/resources/lib/main.py#L279

**Reply by TemofeyF** [reply]
Добрый день! Большое спасибо за уделенное время! К сожалению так и не смог  найти нужный <id фильма/сериала>. 
"play_movie"        :[
      "plugin://video.kino.pub/play/<id фильма/сериала>/"
]
}
вместо <id фильма/сериала> ставил 
{id}, {tmdb}, {imdb}, {tvdb}, {trakt}, {name}, {title}.

**Reply by quarckster** [reply]
Так не будет работать, нужен именно id фильма, который достаётся из REST API кинопаба. Весь плагин на этом и построен, что пользователь видит в интерфейсе названия, но запросы к API посылаются с id. Поиск в плагине есть.

**Reply by TemofeyF** [reply]
а через поиск пробую вот так. не выходит. где я ошибаюсь?

"play_movie"        :[
      "plugin://video.kino.pub/search/all/results?title={title}"
]
}

**Reply by TemofeyF** [reply]
А нельзя по вот этому примеру привязаться к trakt_id?
RunPlugin(plugin://plugin.video.themoviedb.helper?info=play&amp;tmdb_id=$INFO[ListItem.Property(tmdb_id)]&amp;type=movie)
у Вас ведь есть поддержка trakt.tv?

**Comment by TemofeyF** [comment]
а через поиск пробую вот так. не выходит. где я ошибаюсь?

"play_movie" :[
"plugin://video.kino.pub/search/all/results?title={title}"
]
}
